packages/fast-components-styles-msft/src/utilities/high-contrast.ts

```diff
@@ -8,45 +8,75 @@ export const highContrastSelector: string = "@media (-ms-high-contrast:active)";
 
 export enum HighContrastColor {
     text = "WindowText",
-    hyperLinks = "LinkText",
+    forcedColorLink = "LinkText",
+    msLink = "-ms-hotlight",
     disabledText = "GrayText",
     selectedText = "HighlightText",
     selectedBackground = "Highlight",
     buttonText = "ButtonText",
     buttonBackground = "ButtonFace",
-    background = "Background",
+    background = "Window",
 }
 
-// Used to remove text backplate and borders in 'button-text' colors
+// Function used to to opt-out of high contrast color scheme for '-ms' prefixes
+export function applyhighContrastOptOutProperty(): CSSRules<{}> {
```

Comment:
if this function is only being used by the `highContrastOptOutProperty` you can remove the export.

Reply:
export removed